django/kpi/tests.py

```diff
@@ -66,6 +66,10 @@ def test_solutions_kpi_two_snapshot_differences(self):
                                        people_reached=100)
         CountrySolution.objects.create(country=self.country_2, solution=self.sol_1, region=CountryOffice.REGIONS[1][0],
                                        people_reached=23)
+
+        self.assertEqual(Portfolio.objects.get(id=self.port1_resp.json()['id']).landscape_review, False)
```

Comment:
these are not testing the snapshots, just getting the portfolio object. you need to test that the snapshots include the landscape review and see if they change if u change it.
look at `past_snapshot['portfolios'][1]['investment_to_date']` test for example